controllers/topics.js

```diff
@@ -43,9 +43,13 @@ const editTopicHandler = (req, res, next) => {
 		res.render('topic/edit-topic', {
 			action,
 			method,
-			title: req.params.topicId ? 'Thema bearbeiten' : 'Thema anlegen',
-			submitLabel: req.params.topicId ? 'Änderungen speichern' : 'Thema anlegen',
-			closeLabel: 'Abbrechen',
+			title: req.params.topicId
+				? res.$t('topic._topic.headline.editTopic')
+				: res.$t('topic._topic.headline.createTopic'),
+			submitLabel: req.params.topicId
+				? res.$t('topic._topic.button.editTopic')
```

Comment:
$t('topic._topic.button.editTopic') please change it to $t('topic._topic.button.saveChanges')